hell/adventcalendar/2024/13/index.md

```diff
@@ -1,52 +1,68 @@
 
-## Lorem ipsum dolor sit amet.
+<img src="google-translate-popup.png" width="800" height="161" loading="lazy" alt="A screenshot from Google Translate. It displays language selection options, with Dutch on the left and English on the right. English is selected as the target language.">
 
-Lorem ipsum dolor sit amet `consectetur` adipisicing elit. In aperiam alias laborum magnam accusamus accusantium vero. Exercitationem distinctio at quisquam quibusdam rem voluptatum illo, laboriosam natus aperiam sequi, voluptate inventore!
+Browser auto-translation relies on the lang attribute set on the HTML element:
 
+```html
+<html lang="en"></html>
+```
+
+However, automatic translations can sometimes be inaccurate or misleading, especially when it comes to proper names, email addresses, branded names, or technical terms that should remain untranslated.
+
+For example, let’s see what happens when a user’s name, like _Joke_ (a Dutch feminine name pronounced [ˈjoːkə]), is displayed on an English page that is then auto-translated into Dutch:
 
 ```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-	  <meta charset="utf-8" />
-	  <title>Webpage starter</title>
-  </head>
-  <body>
-    <h1>Hello World!</h1>
-  </body>
-</html>
+<!-- Original content in English -->
+<p>Name: Joke</p>
+<p>Email: joke@test.com</p>
+
+<!-- Translated content in Dutch -->
+<p>Naam: Grap</p>
+<p>E-mailadres: grap@test.com</p>
 ```
 
-## Lorem ipsum dolor sit amet.
+The Dutch word _“Grap”_ means _“joke”_ — a thing said to cause amusement — turning the name into something unintended and causing confusion.
+
+To prevent such misinterpretations, we can use the <code>[translate](https://html.spec.whatwg.org/multipage/dom.html#the-translate-attribute)</code> attribute, which indicates whether an element content should or should not be translated. The attribute can appear on any element, and it takes just two values: `yes` or `no`. In our case, we need to use the value `no` to disable automatic translation for specific pieces of content:
 
-* Lorem, ipsum.
-* Lorem, ipsum.
-* Lorem, ipsum.
-* Lorem, ipsum.
+```html
+<!-- Original content in English -->
+<p>Name: <span translate="no">Joke</span></p>
+<p>Email: <span translate="no">joke@test.com</span></p>
+
+<!-- Translated content in Dutch -->
+<p>Naam: <span translate="no">Joke</span></p>
+<p>E-mailadres: <span translate="no">joke@test.com</span></p>
+```
```

Comment:
I was thinking about giving even more examples and maybe going deeper into which elements are not auto-translated...

For now, I have focused on a "snippet" idea and making it short. Let me know if additional info and context would be helpful!